docs/DeveloperGuide.adoc

```diff
@@ -109,27 +110,26 @@ Given below is the Sequence Diagram for interactions within the `Logic` componen
 .Interactions Inside the Logic Component for the `delete 1` Command
 image::DeleteSequenceDiagram.png[]
 
-NOTE: The lifeline for `DeleteCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
+NOTE: The lifeline for `DeleteAddressCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
 
 [[Design-Model]]
 === Model component
 
 .Structure of the Model Component
-image::ModelClassDiagram.png[]
+image::ModelDiagram.png[]
```

Comment:
We feel that there is no need to show lower-level details in your architecture diagram. We suggest that the lowest-level details in this diagram should be AddressBook, Planner, etc. Anything lower should be shown as their own class diagrams (e.g. PlannerClassDiagram) in later sections of the DG.

Also, we are confused over why Model implements AddressModel, UserPrefsModel, etc. Wouldn't doing it the other way around make more sense according to OOP? Or maybe it is just a naming issue that is causing confusion (maybe renaming Model as JarvisModel would make it feel more natural since 'Model' feels like a baseline interface).